frontend/src/components/ingress/Details.tsx

```diff
@@ -1,12 +1,157 @@
+import { Box, Link, Typography } from '@material-ui/core';
 import { useTranslation } from 'react-i18next';
 import { useParams } from 'react-router-dom';
 import Ingress, { IngressRule } from '../../lib/k8s/ingress';
+import { IngressBackend } from '../../lib/k8s/ingress';
 import { useSettings } from '../App/Settings/hook';
 import LabelListItem from '../common/LabelListItem';
 import { DetailsGrid } from '../common/Resource';
 import { SectionBox } from '../common/SectionBox';
 import SimpleTable from '../common/SimpleTable';
 
+/**
+ * This function is used to determine the protocol used in the formatURL for LinkedStringFormat
```

Comment:
How about this?

```suggestion
Is https used in the ingress item?
```

"This function is used to" is a bit redundant.